### PR TITLE
Fix linux-presto-fuzzer-run CI build failure

### PR DESCRIPTION
**Pull request by majetideepak**

### CODE REVIEW

**Review by assignUser — APPROVED** [review]
That should do it :)

**Comment by pedroerp** [comment]
Can you comment on why we need to explicitly specify we want DuckDB bundled? Is it because sometimes there is DuckDB already installed in the system?

**Review by kgpai — APPROVED** [review]

**Comment by assignUser** [comment]
@pedroerp on that job we force the use of system dependencies and ddb is not installed in that image (something we can fix in a follow up)

File: .circleci/dist_compile.yml [hunk 1]
```diff
@@ -460,6 +460,7 @@ jobs:
     environment:
       VELOX_DEPENDENCY_SOURCE: SYSTEM
       simdjson_SOURCE: BUNDLED
+      DuckDB_SOURCE: BUNDLED
```
Comment (mbasmanova) [inline .circleci/dist_compile.yml:463]: Why are we using BUNDLED and not SYSTEM here. 

I believe this change (inadvertently) broken join fuzzer CI runs: https://github.com/facebookincubator/velox/issues/7943

CC: @kgpai

**Comment by mbasmanova** [comment]
> Can you comment on why we need to explicitly specify we want DuckDB bundled? Is it because sometimes there is DuckDB already installed in the system?

I wish we fixed that. It would have save me long hours debugging https://github.com/facebookincubator/velox/issues/7943 which happens only if DuckDB is build in debug mode.

**Pull request closed without merging**